### DISCUSSION TITLE
Jinja2 vs Jinja3, why do all references for v3 go to Jinja2?

### DISCUSSION THREAD

**Discussion by mikaello** [pallets/jinja#1899] [Q&A] [answered]
When I go to the Jinja website, I notice in the URL that I am browsing version 3 of the docs, but when clicking the PyPI release I only get the jinja2-release, why is that? And when searching, I can't find a jinja3 PyPi release, only Jinja2. What is the relationship between Jinja2 and Jinja3?

![image](https://github.com/pallets/jinja/assets/2505178/bdb649f4-a910-43bf-964e-452c5a7dee18)

**Comment by ThiefMaster** [comment, marked as answer]
The package name is Jinja2 for historical reasons. So it's Jjinja2 version 3 ;)

**Reply by tenstad** [reply]
Could you please elaborate a bit on _historical reasons_? :D

**Reply by ThiefMaster** [reply]
last jinja (not jinja2) release was in 2007. before my time, probably only @mitsuhiko remembers the details. maybe it was to allow coexistence of jinja1 and jinja2 at some point.
